features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
```

Comment:
Is there a hidden newline or something? I'm getting this when I try to repeat in irb:
```ruby
jruby-1.7.19 :001 > a = " IT / "
 => " IT / " 
jruby-1.7.19 :002 > a.split('/').collect{ |x| x.nil? ? '' : x.strip }
 => ["IT", ""] 
jruby-1.7.19 :003 > a.split('/')
 => [" IT ", " "] 
jruby-1.7.19 :004 > a = " IT /"
 => " IT /" 
jruby-1.7.19 :005 > a.split('/')
 => [" IT "] 
jruby-1.7.19 :006 > a.split('/').collect{ |x| x.nil? ? '' : x.strip }
 => ["IT"]
```